components/brave_shields/browser/ad_block_service.cc

```diff
@@ -28,6 +28,8 @@
 #include "components/prefs/pref_service.h"
 #include "net/base/registry_controlled_domains/registry_controlled_domain.h"
 
+#include "base/json/json_writer.h"
```

Comment:
I think maybe I added this when I was working on changes and then forgot to delete it or if it's needed move it to the correct place